llvm/test/tools/sycl-post-link/emit_imported_symbols.ll

```diff
@@ -0,0 +1,67 @@
+; This test checks that the -emit-imported-symbols option generates a list of imported symbols
+;
+; RUN: sycl-post-link -symbols -emit-imported-symbols -split=kernel -S < %s -o %t.table
+
+; RUN: FileCheck %s -input-file=%t_0.sym --check-prefixes CHECK-SYM-0
+; RUN: FileCheck %s -input-file=%t_1.sym --check-prefixes CHECK-SYM-1
+; RUN: FileCheck %s -input-file=%t_2.sym --check-prefixes CHECK-SYM-2
+
+;; Function names were chosen so that no function with a 'inside' in their function name is imported
+; RUN: FileCheck %s -input-file=%t_0.prop --check-prefixes CHECK-IMPORTED-SYM-0 --implicit-check-not='inside'
+; RUN: FileCheck %s -input-file=%t_1.prop --check-prefixes CHECK-IMPORTED-SYM-1 --implicit-check-not='inside'
+; RUN: FileCheck %s -input-file=%t_2.prop --check-prefixes CHECK-IMPORTED-SYM-2 --implicit-check-not='inside'
+
+; CHECK-SYM-0: middle
```

Comment:
Why do we need to check sym files here? Not complaining. But trying to understand if it is relevant to this change. Thanks

Reply:
Checking the sym files is sort of just a comment so that it is clear what the entry point for an image is.   This makes it easier to understand what the imported symbols should be.